src/main/java/uiPages/ProductPage.java

```diff
@@ -0,0 +1,70 @@
+package uiPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import helpers.WebElementHelper;
+
+import static helpers.WebElementHelper.areVisible;
+import static helpers.WebElementHelper.setFieldValue;
+
+public class ProductPage extends BasePage {
+
+    private static final String URL_PATH = "/product";
+
+    @FindBy(className = "product-essential__description")
+    private WebElement productDescriptionField;
+
+    @FindBy(className = "product-essential__specification")
+    private WebElement productSpecificationField;
+
+    @FindBy(xpath = "//button[@class='single_add_to_cart_button button alt disabled wc-variation-selection-needed']")
+    private WebElement disabledAddToCart;
+
+    @FindBy(xpath = "//*[@id=\'pa_color\']/option[2]")
+    private WebElement attributeColor;
+
+    @FindBy(xpath = "//*[@id=\'pa_size\']/option[2]")
+    private WebElement attributeSize;
+
+    @FindBy(className = "woocommerce-message")
+    private WebElement addedmessage;
+
+    @FindBy(xpath = "//button[@class='single_add_to_cart_button button alt']")
+    private WebElement addToCart;
```

Comment:
disabledAddToCart and addToCart are the same element. They need to exist as a single WebElement. 
Refer to the disabledAddToCart review comment.

Reply:
done